mod/photos.php

```diff
@@ -914,7 +914,19 @@ function photos_content(App $a)
 			'$submit' => DI::l10n()->t('Submit'),
 		]);
 
-		$usage_message = '';
+		// Determine which setting actually limits  upload size of images: limit of Friendica ('maximagesize') or upload_max_filesize
+		$maximagesize_Mbytes = 0;
+		// Get the relevant size limits for uploads. Abbreviated var names: MaxImageSize -> mis; upload_max_filesize -> umf
+		$mis_bytes = DI::config()->get('system', 'maximagesize');
+		$umf_bytes = Strings::getBytesFromShorthand(get_cfg_var('upload_max_filesize'));
+
+		if (is_numeric($mis_bytes)) {
```

Comment:
This shouldn't prevent `$maximagesize_Mbytes` value to be set. It should prevent `$mis_bytes` to be used to set its value though, so I would add it to the condition that compares the numbers.